pull_request_template.md

```diff
@@ -1,6 +1,23 @@
 # Purpose of this PR
 
+
+## Type of change
+
+- [ ] Bug fix 
+- [ ] Refactoring
+- [ ] New feature 
+- [ ] Minor change
+- [ ] Fundamental change
+- [ ] This change requires a documentation update
+
+
 ## Checklist:
-* updated in code documentation
-* adjusted reporting if needed
-* checked that REMIND still compiles
+
+- [ ] My code follows the coding ettiquette
+- [ ] I have performed a self-review of my own code
+- [ ] Changes are commented, particularly in hard-to-understand areas
+- [ ] I have updated the in code documentation
+- [ ] I have adjusted reporting where it was needed
+- [ ] The model compiles and starts successfully
```

Comment:
I usually added the path to the run so reviewers can have a look at the log if they have a suspicion.

Reply:
sometimes, user make local tests and external users will not be able to provide a path for everyone. I will add the option for a path to test runs

Reply:
What is the "default" scenario? Starting REMIND with the untouched default.cfg? I am not sure (1) that all people know this (2) that people will do this test runs before. They will probably have a test run with some configuration where they check the effects of their changes. Should they always to a "default" run?

Reply:
true, "default" might also change over time....in future, I expect that we will have a predefined set of test-scenarios. For now I would hope that developer to test runs - at least for SSP2 - if they introduce new features or make fundamental changes